config/application.rb

```diff
@@ -218,7 +218,7 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :dynflow
 
     Foreman::Logging.configure(
-      :log_directory => "#{Rails.root}/log",
+      :log_directory => SETTINGS.dig(:logging, :log_directory) || "#{Rails.root}/log",
```

Comment:
Systemd also exposes `$LOGS_DIRECTORY` when using `LogsDirectory=` in the unit file. Should we use/prefer that?